lib/src/schema/extensions/value_transfer_transaction.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:witnet/schema.dart';
+
+extension Metadata on VTTransaction {
+  List<PublicKeyHash> get metadata {
+    List<PublicKeyHash> metadata = this
+        .body
+        .outputs
+        .where((ValueTransferOutput output) => output.value == 1)
```

Comment:
is the metadata only supposed to be a single output in the 0 index?
what happens when a user pays with a 1 nWIT fee?